pkg/mountutil/mountutil.go

```diff
@@ -54,7 +54,7 @@ func ProcessFlagV(s string, volStore volumestore.VolumeStore) (*Processed, error
 		dst = s
 		res.AnonymousVolume = idgen.GenerateID()
 		logrus.Debugf("creating anonymous volume %q, for %q", res.AnonymousVolume, s)
-		anonVol, err := volStore.Create(res.AnonymousVolume)
+		anonVol, err := volStore.Create(res.AnonymousVolume, []string{})
```

Comment:
Compilation is failing, not sure why though

```
#15 61.55 # github.com/containerd/nerdctl/pkg/mountutil
#15 61.55 pkg/mountutil/mountutil.go:72:32: not enough arguments in call to volStore.Create
#15 61.55 	have (string)
#15 61.55 	want (string, []string)
```
https://github.com/containerd/nerdctl/pull/274/checks?check_run_id=2984458522#step:3:1290

Reply:
@ktock @fuweid Could you take a look?

I don't see `volstore.Create()` in `mountutil.go:72`, not sure why this is failing

Reply:
@fahedouch This PR needs rebase.

https://github.com/containerd/nerdctl/blob/fde3eae7cc21f6b010e683026837cab11d544f9f/pkg/mountutil/mountutil.go#L72

Reply:
yep thank you @ktock